vagrant/setup.sh

```diff
@@ -30,6 +30,24 @@ Argument:
 EOF
 }
 
+function enable_iommu {
+    echo "Checking Intel iommu"
+    if ! find /sys/kernel/iommu_groups/*/devices/* > /dev/null 2>&1; then
+        echo "Enabling Intel_iommu"
+        sudo sed -i "s|^GRUB_CMDLINE_LINUX\(.*\)\"|GRUB_CMDLINE_LINUX\1 intel_iommu=on\"|g" /etc/default/grub
```

Comment:
Are all distros grub based (still)? Clear isn't but it comes with IOMMU enabled by default.

Reply:
The line 144 of this file specifies the use case only for rhel|centos|fedora distributions

Reply:
of course. I lost the track that this was for that case only. thanks!